pom.xml

```diff
@@ -40,6 +40,7 @@
 
   <properties>
      <jenkins.version>1.625.3</jenkins.version>
+     <workflow.version>1.8</workflow.version>
```

Comment:
what's this for? i don't see it used in this file.  Unless jenkins framework is using it in some special way?